test/e2e-test-application/src/app/project/project.component.html

```diff
@@ -517,6 +517,23 @@ <h3 class="fd-layout-panel__title">Navigate</h3>
             >
             <app-code-snippet data="linkManager().goBack()"> </app-code-snippet>
           </li>
+          <li class="fd-list__item">
+            <a
+              href="javascript:void(0)"
+              class="fd-link"
+              (click)="
+                linkManager().navigate(
+                  '/#?Intent=Sales-settings?param1=abc&param2=bcd'
+                )
+              "
```

Comment:
Semantically, we should use `button` element for handling onClick event, not an anchor tag. 
I knew we applied click event on other anchor tag in this file. Maybe you can leave it now, we do refactoring in a follow-up ticket.

Reply:
agree 👍 , a follow up ticket could be good, although I wouldn't give it high priority